src/main/java/uk/gov/hmcts/darts/audio/service/AudioService.java

```diff
@@ -13,6 +15,8 @@ public interface AudioService {
 
     InputStream preview(Integer mediaId);
 
+    Flux<ResponseEntity<byte[]>> getAudioPreviewFlux(Integer mediaId, String range);
```

Comment:
May be worth keep the flux calls in a seperate place/package just to allow them to be easily identifiable

Reply:
I think it's okay here as it's still related to the audio service.

Reply:
yeah agreed. just a suggestion